Documentation/signing-and-verification-guide.md

```diff
@@ -143,7 +143,7 @@ pubkeys.gpg
 
 ## Distributing Images via Meta Discovery
 
-Host an HTML page with the following meta tags:
+Host `example.com/hello` with the following HTML contents and meta tags:
```

Comment:
This still does not inform under which url this html contents should be accessible.
But I agree - it's more readable than original.